code/modules/client/preferences.dm

```diff
@@ -106,6 +106,8 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	var/list/menuoptions
 
+	var/fancy_chat = TRUE

```

Comment:
This will actually default to FALSE because default preference values don't work for some reason. Maybe invert the logic?